tests/test_backup.py

```diff
@@ -367,6 +367,145 @@ def test_delete_backup(self, mock_available_backups, tmpdir, caplog):
         assert deleted is True
         assert child_backup.backup_id not in parent_backup.children_backup_ids
 
+        # Test 8: Update next rsync backup information
+        given_backup = build_test_backup_info(
+            backup_id="rsync_backup_id",
+            server=backup_manager.server,
+        )
+        build_backup_directories(given_backup)
+        next_backup = build_test_backup_info(
+            backup_id="next_rsync_backup_id",
+            server=backup_manager.server,
+        )
+        build_backup_directories(next_backup)
+        mock_available_backups.return_value = {
+            given_backup.backup_id: given_backup,
+            next_backup.backup_id: next_backup,
+        }
+        with patch("barman.backup.BackupManager.get_next_backup") as get_next_backup:
+            with patch(
+                "barman.backup.BackupManager._set_backup_sizes"
+            ) as set_backup_sizes:
+                get_next_backup.return_value = next_backup
+                deleted = backup_manager.delete_backup(given_backup)
+                assert deleted is True
+                set_backup_sizes.assert_called_with(next_backup)
```

Comment:
```suggestion
                set_backup_sizes.assert_called_once_with(next_backup)
```

I think "once" provides a more precise meaning here.